pdr_backend/lake/persistent_data_store.py

```diff
@@ -127,7 +129,7 @@ def insert_to_table(self, df: pl.DataFrame, table_name: str):
             return
 
         logger.info("create_and_fill_table = %s", table_name)
-        logger.info("create_and_fill_table DF = %s", df)
+        logger.info("\n%s", df)
```

Comment:
no spacing like this in logs, please. Empty spaces are for design purposes, not for arrangement. In the worst case, there are similar instructions in predictoor and trader, where IF the logging has stdout, it prints the spacings. Please implement the same logic here, if needed.

ref: https://github.com/oceanprotocol/pdr-backend/blob/ca3e40a508ab3f4facbe6c360ff442996e1745eb/pdr_backend/util/logutil.py#L4